specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/preview/2020-07-01-preview/examples/AzureADMetricsCreate.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "rg1",
+    "azureADMetricsName": "ddb1",
+    "api-version": "2020-07-01-preview",
+    "azureADMetricsConfig": {
+      "name": "myOrgAzureADMetrics"
```

Comment:
>"name": "myOrgAzureADMetrics" [](start = 6, length = 29)

"name" is passed in above in azureADMetricsName.  The only thing that would be passed in here is tags